scripts/rook.sh

```diff
@@ -33,57 +34,67 @@ rook_version() {
-
-            curl -o "${TEMP_DIR}"/cluster-test.yaml "${ROOK_URL}/cluster-test.yaml"
-            sed -i "s|image.*|${ROOK_CEPH_CLUSTER_VERSION_IMAGE_PATH}|g" "${TEMP_DIR}"/cluster-test.yaml
-            sed -i "s/config: |/config: |\n    \[mon\]\n    mon_warn_on_insecure_global_id_reclaim_allowed = false/g" "${TEMP_DIR}"/cluster-test.yaml
-            sed -i "s/healthCheck:/healthCheck:\n    livenessProbe:\n      mon:\n        disabled: true\n      mgr:\n        disabled: true\n      mds:\n        disabled: true/g" "${TEMP_DIR}"/cluster-test.yaml
-			cat  "${TEMP_DIR}"/cluster-test.yaml
-            kubectl_retry create -f "${TEMP_DIR}/cluster-test.yaml"
-        fi
-        rm -rf "${TEMP_DIR}"
-
-        kubectl_retry create -f "${ROOK_URL}/toolbox.yaml"
-        kubectl_retry create -f "${ROOK_URL}/filesystem-test.yaml"
-        kubectl_retry create -f "${ROOK_URL}/pool-test.yaml"
-
-        # Check if CephCluster is empty
-        if ! kubectl_retry -n rook-ceph get cephclusters -oyaml | grep 'items: \[\]' &>/dev/null; then
-            check_ceph_cluster_health
-        fi
-
-        # Check if CephFileSystem is empty
-        if ! kubectl_retry -n rook-ceph get cephfilesystems -oyaml | grep 'items: \[\]' &>/dev/null; then
-            check_mds_stat
-        fi
-
-        # Check if CephBlockPool is empty
-        if ! kubectl_retry -n rook-ceph get cephblockpools -oyaml | grep 'items: \[\]' &>/dev/null; then
-            check_rbd_stat ""
-        fi
+	kubectl_retry create -f "${ROOK_URL}/common.yaml"
+
+	ROOK_MAJOR=$(rook_version 1)
+	ROOK_MINOR=$(rook_version 2)
+
+	# If rook version is > 1.5 , we will apply CRDs.
+	if [ "${ROOK_MAJOR}" -eq 1 ] && [ "${ROOK_MINOR}" -ge 5 ]; then
+		kubectl_retry create -f "${ROOK_URL}/crds.yaml"
+	fi
+	TEMP_DIR="$(mktemp -d)"
+	curl -o "${TEMP_DIR}/operator.yaml" "${ROOK_URL}/operator.yaml"
+	# disable rook deployed csi drivers
+	sed -i 's|ROOK_CSI_ENABLE_CEPHFS: "true"|ROOK_CSI_ENABLE_CEPHFS: "false"|g' "${TEMP_DIR}/operator.yaml"
+	sed -i 's|ROOK_CSI_ENABLE_RBD: "true"|ROOK_CSI_ENABLE_RBD: "false"|g' "${TEMP_DIR}/operator.yaml"
+
+	kubectl_retry create -f "${TEMP_DIR}/operator.yaml"
+	# Override the ceph version which rook installs by default.
+	if [ -z "${ROOK_CEPH_CLUSTER_IMAGE}" ]; then
+		kubectl_retry create -f "${ROOK_URL}/cluster-test.yaml"
+	else
+		ROOK_CEPH_CLUSTER_VERSION_IMAGE_PATH="image: ${ROOK_CEPH_CLUSTER_IMAGE}"
+
+		curl -o "${TEMP_DIR}"/cluster-test.yaml "${ROOK_URL}/cluster-test.yaml"
+		sed -i "s|image.*|${ROOK_CEPH_CLUSTER_VERSION_IMAGE_PATH}|g" "${TEMP_DIR}"/cluster-test.yaml
+		sed -i "s/config: |/config: |\n    \[mon\]\n    mon_warn_on_insecure_global_id_reclaim_allowed = false/g" "${TEMP_DIR}"/cluster-test.yaml
+		sed -i "s/healthCheck:/healthCheck:\n    livenessProbe:\n      mon:\n        disabled: true\n      mgr:\n        disabled: true\n      mds:\n        disabled: true/g" "${TEMP_DIR}"/cluster-test.yaml
```

Comment:
```suggestion
		sed -i "s/healthCheck:/healthCheck:\n    livenessProbe:\n      mon:\n        disabled: true\n      mgr:\n        disabled: true\n      mds:\n        disabled: true\n    startupProbe:\n      mon:\n        disabled: true\n      mgr:\n        disabled: true\n      mds:\n        disabled: true/g" "${TEMP_DIR}"/cluster-test.yaml
```

Reply:
added a commit, for now, let's see. if CI passes will update last commit to fix DCO and commitlint